DevCycleTests/Networking/DevCycleServiceTests.swift

```diff
@@ -81,6 +81,12 @@ class DevCycleServiceTests: XCTestCase {
         let config = processConfig(data)
         XCTAssertNil(config)
     }
+    
+    func testProcessConfigReturnsErrorIfInvalidJson() throws {
+        let data = "{".data(using: .utf8)
+        let config = processConfig(data)
+        XCTAssertNil(config)
```

Comment:
should we add an assertion to check that it throws the `InvalidJson` error here?

Reply:
it gets caught and logged, not thrown upwards unfortunately